config/default/filter.format.filtered_html.yml

```diff
@@ -71,20 +73,20 @@ filters:
     status: true
     weight: -49
     settings:
-      allowed_html: '<a href hreflang title class id target aria-label rel data-*> <em> <strong> <cite> <blockquote class cite> <code> <ul type class> <ol start type class> <li> <dl> <dt> <dd> <h2 id class> <h3 id class> <h4 id class> <h5 id class> <h6 id class> <s> <sup> <sub> <p class> <pre class> <div class> <span class> <hr class> <table class border summary id role="presentation, grid" aria-label aria-labelledby> <caption id> <tbody class> <thead class> <tfoot class> <th id align valign colspan rowspan scope class> <td align valign colspan rowspan class headers> <tr align valign class> <drupal-media data-entity-type data-entity-uuid data-align data-caption data-view-mode title alt> <br> <mark> <small class> <iframe src width height allowfullscreen title loading="lazy" seamless sandbox="allow-same-origin allow-scripts allow-popups">'
+      allowed_html: '<em> <strong> <cite> <blockquote cite class=""> <code> <ul type class=""> <ol start type class=""> <li> <dl> <dt> <dd> <s> <sup> <sub> <pre class=""> <span class=""> <hr class=""> <table border summary id role aria-label aria-labelledby class=""> <caption id> <tbody class=""> <thead class=""> <tfoot class=""> <th id align valign colspan rowspan scope class=""> <td align valign colspan rowspan headers class=""> <tr align valign class=""> <drupal-media data-entity-type data-entity-uuid data-align data-caption data-view-mode title alt> <br> <mark> <small class=""> <iframe src width height allowfullscreen title loading seamless sandbox> <a href hreflang title id target aria-label rel data-* class=" bttn bttn--caps bttn--primary bttn--secondary"> <p class=" uids-component--bold-intro uids-component--light-intro"> <h2 id class=" headline headline--serif headline--underline"> <h3 id class=" headline headline--serif headline--underline"> <h4 id class=" headline headline--serif headline--underline"> <h5 id class=" headline headline--serif headline--underline"> <h6 id class=" headline headline--serif headline--underline"> <div class=" alert alert-success alert-info alert-warning alert-danger">'
```

Comment:
Why are things like `uids-component--bold-intro uids-component--light-intro` these being hard-coded here?

Reply:
Same for `headline headline--serif headline--underline`

Reply:
I had this issue with a separate PR. Any time a change gets made and re-exported, I think it tries to consolidate and restrict.

There's probably a way to do it without it adding extra changes, but for mine I ended up just manually editing the file and adding the html needed.

Reply:
Thanks, Cory. Doing just that. Only adding what's needed.

Reply:
Reverting, cr didn't fix the issue.

Reply:
I looked at files changed here (https://github.com/uiowa/uiowa/pull/5276/commits/5cd5b0ee6c5d430a9ddad3ec85f9ad4fba6bfdc7) and that filtered_html is not being set by the module.

Reply:
Cory's commit and importing config has resolved the issue.